sys/openbsd/init.go

```diff
@@ -91,6 +93,13 @@ func (arch *arch) SanitizeCall(c *prog.Call) {
 		for _, f := range badflags {
 			flags.Val &= ^f
 		}
+	case "ioctl":
+		// Performing the following ioctl on a /dev/pf file descriptor
```

Comment:
Does ioctl have a single namespace? I.e. is the value of DIOCKILLSTATES ever indicate anything else for a different kind of file descriptor? Maybe drop a comment indicating that this doesn't wipe out too much coverage?

Reply:
> Does ioctl have a single namespace? I.e. is the value of DIOCKILLSTATES ever indicate anything else for a different kind of file descriptor?

They do. Right now, no other ioctl command currently supported by syzkaller collide with DIOCKILLSTATES (3235922985). The risk is very low according to my understanding but there's no guarantee for uniqueness.

Reply:
Maybe note this down in the comment such that somebody looking at the code doesn't have to think this through? Also the fact about 0 being invalid below.

Reply:
done